### PR TITLE
Guide: Remotely configure RUM using LaunchDarkly

### PR DESCRIPTION
**Pull request by meghanlo**
### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by kayayarai** [comment]
Opened https://datadoghq.atlassian.net/browse/DOCS-4444 to track editorial review

File: content/en/real_user_monitoring/guide/remotely-configure-rum-using-launchdarkly.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+  text: 'Visualize your RUM data in the RUM Explorer'
+---
+
+## Overview
+When instrumenting a [RUM application][1], you can remotely control the RUM initialization configurations based on your immediate needs, such as an ongoing incident where you might need higher fidelity data.
+
+Instead of having to deploy the changes to your RUM initialization configurations, you can use feature flags. Feature flag management companies like [LaunchDarkly][2] evaluate feature flags on the server side and thus allow you to make changes to your code without needing to redeploy it. 
+
+## Setting up your flag in LaunchDarkly
+To set up your flag in LaunchDarkly, start by following their documentation on [setting up an SDK][3]. More information on the Client-side SDK is available in LaunchDarkly's [documentation][4].
```
Comment (rtrieu) [inline content/en/real_user_monitoring/guide/remotely-configure-rum-using-launchdarkly.md:21]:
```suggestion
To set up your flag in LaunchDarkly, start by following their documentation on [setting up an SDK][3]. For additional details, see LaunchDarkly's [Client-side SDK documentation][4].
```

File: content/en/real_user_monitoring/guide/remotely-configure-rum-using-launchdarkly.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+```
+
+However, before you can pass in the configuration to initialize RUM, you'll need to create an Object from the flag JSON value. Once you've done that, you can initialize the RUM SDK.
+
+```javascript
+datadogRum.init(RUM_configuration_object)
+```
+
+## Embed LaunchDarkly's controls to configure RUM directly in your Dashboards
+If you want to change your RUM configuration directly in your Datadog application, you can embed the LaunchDarkly UI into Datadog and switch your feature flag on/off. The feature flags are set up so you can keep them off, with the default values. When you want to have higher fidelity data, you can turn on your feature flag and the values you set for the ON variation will now be used for the RUM initialization.
```
Comment (rtrieu) [inline content/en/real_user_monitoring/guide/remotely-configure-rum-using-launchdarkly.md:118]:
```suggestion
If you want to change your RUM configuration directly in your Datadog application, you can embed the LaunchDarkly UI into Datadog and switch your feature flag on/off. The feature flags are set up so you can keep them off, with the default values. When you want to have higher fidelity data, you can turn on your feature flag and the values you set for the ON variation will be used for the RUM initialization. 
```

**Review by rtrieu — CHANGES_REQUESTED** [review]
Requesting another review to make sure it all looks good, then we can merge!

**Review by rtrieu — APPROVED** [review]

**Pull request merged**